src/Makefile.am

```diff
@@ -6,7 +6,7 @@
 print-%: FORCE
 	@echo '$*'='$($*)'
 
-DIST_SUBDIRS = secp256k1 univalue
+DIST_SUBDIRS = secp256k1 minisketch univalue
```

Comment:
Pretty sure there's no need for this since we're building it ourselves. IIRC including it here was problematic while I was testing, but I've forgotten why.

Reply:
Yea this isn't needed, and was breaking `make dist`. Have dropped it from my branch.